### DISCUSSION TITLE
whats zero ?

### DISCUSSION THREAD

**Discussion by parweb** [onejs/one#117] [Q&A] [answered]
I saw the talk on vite conf about one but i didnt understood whats zero ?
an orm ? where i can find more informations ?

**Comment by jessemillman** [comment, marked as answer]
Zero is the local-first sync layer, by the same creators of replicache. Still in the very early stages, but you can learn more here: https://zerosync.dev/
If you're super unfamiliar with local first, definitely check out the replicache docs (https://replicache.dev/) - will help more with the fundamental concepts.
